ush/python_utils/filesys_cmds_vrfy.py

```diff
@@ -5,13 +5,13 @@
 
 
 def cmd_vrfy(cmd, *args):
-    """Execute system command
+    """Executes system command
 
     Args:
-        cmd: the command
-        *args: its arguments
+        cmd (str): The command
+        *args: Its arguments
     Returns:
-        Exit code
+        ret - Exit code
```

Comment:
Should the hyphen be replaced with a colon?

```suggestion
        ret: Exit code
```